src/init.cpp

```diff
@@ -1249,10 +1243,10 @@ bool AppInitMain(InitInterfaces& interfaces)
             // and because this needs to happen before any other debug.log printing
             LogInstance().ShrinkDebugFile();
         }
-        if (!LogInstance().OpenDebugLog()) {
-            return InitError(strprintf("Could not open debug log file %s",
+    }
+    if (!LogInstance().StartLogging()) {
+        return InitError(strprintf("Could not open debug log file %s",
```

Comment:
Could this message be translated?
Ref: [Translation Strings Policy](https://github.com/bitcoin/bitcoin/blob/master/doc/translation_strings_policy.md#gui-strings)

Reply:
Leaving it to minimize the diff